region/maridia/inner-yellow.json

```diff
@@ -168,6 +168,54 @@
                     "It is possible to spark through the doorway, but diagonal sparks are not supported.",
                     "There is no gravity version, as Gravity + Speedbooster is enough to get up."
                   ]
+                },
+                {
+                  "name": "Cross Room Jump - Speed Jump to the Top",
+                  "notable": false,
+                  "requires": [
+                    "canCrossRoomJumpIntoWater",
+                    {"adjacentJumpway": {
+                      "jumpwayType": "platformBelow",
+                      "fromNode": 2,
+                      "minHeight": 9,
+                      "maxHeight": 9,
+                      "maxLeftPosition": -38.5,
+                      "minRightPosition": -7
+                    }},
+                    "SpeedBooster",
```

Comment:
I clarified the note now to explain the setup: you want to start as far to the left as possible (hanging over the edge as far as possible). The trickyDashJump is easier, but from what I can tell it can't be used unless the Power Bomb blocks are cleared since it requires a jump point further to the right. We could add another strat for that case but not sure it would be worth it, since as far as the logic goes it's already covered by this case